test/unit/tools/HTMLImportTranscoder/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Hello, World!</title>
+    <script src="../../../../bin/traceur.js"></script>
+    <script src="../../../../src/bootstrap.js"></script>
+    <script src="../../../../third_party/webcomponentsjs/webcomponents-lite.js"></script>
+    <script>
+        $traceurRuntime.options.experimental = true;
```

Comment:
It's a copy from [test/unit/system/ScriptTypeTextTraceur.html](https://github.com/google/traceur-compiler/blob/master/test/unit/system/ScriptTypeTextTraceur.html)

I will remove it